src/lib/gates.ts

```diff
@@ -42,6 +43,24 @@ function rangeCheck64(x: Field) {
   );
 }
 
+function rot(
+  field: Field,
+  rotated: Field,
+  excess: Field,
+  limbs: [Field, Field, Field, Field],
+  crumbs: [Field, Field, Field, Field, Field, Field, Field, Field],
+  two_to_rot: Field
```

Comment:
since this is a constant gate parameter, I think it should be passed as a bigint (to avoid ambiguity with variables),
```suggestion
  two_to_rot: bigint
```

Reply:
fixed in https://github.com/o1-labs/o1js/pull/1182/commits/00530afef51180d9e10fac8a619c7c6c478597d3